src/Components/Aspire.Confluent.Kafka/HealthCheckExtensions.cs

```diff
@@ -0,0 +1,32 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+using Microsoft.Extensions.Hosting;
+
+namespace Aspire.Confluent.Kafka;
+
+internal static class HealthCheckExtensions
```

Comment:
You shouldn't need to copy this file. You can just source link to the `Common` folder.

Reply:
done